guardian/scribe-plugin-noting
# Zero width spaces at the end of paragraphs

This PR adds zero width spaces to the end of any paragraph in which a note spans to the end. This is a direct fix for https://github.com/guardian/scribe-plugin-noting/issues/90

### QA
- Make a few paragraphs containing notes that span to the end. Ensure that you can click after the note and that the caret is placed within the paragraph but after the note.
